include/oneapi/tbb/concurrent_queue.h

```diff
@@ -261,7 +263,9 @@ class concurrent_bounded_queue {
     template <typename FuncType>
     void internal_wait(r1::concurrent_monitor* monitors, std::size_t monitor_tag, std::ptrdiff_t target, FuncType pred) {
         d1::delegated_function<FuncType> func(pred);
+#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 1070
         r1::wait_bounded_queue_monitor(monitors, monitor_tag, target, func);
```

Comment:
What is the issue with calling `wait` here?

Reply:
That was an issue with different export name, I think. Fixed now, macro dropped.